hooks/pre-exit

```diff
@@ -2,13 +2,17 @@
 
 set -euo pipefail
 
-nvm_installation_dir=$(buildkite-agent meta-data get "automattic-nvm-installation-dir")
-# if file $nvm_dir/.automattic-nvm-dir-marker" exists
-if [[ -f "$nvm_installation_dir/.automattic-nvm-dir-marker" ]]; then
+# Use the same deterministic directory path as in pre-command
+TEMP_DIR="${TMPDIR:-/tmp}"
+nvm_dir="${TEMP_DIR}/automattic-nvm-${BUILDKITE_JOB_ID}"
+
+# Check if the directory exists and has our marker file
+if [[ -d "$nvm_dir" ]]; then
     echo "~~~ :node: Uninstalling Node.js"
-    echo "Removing $nvm_installation_dir"
-    rm -rf "$nvm_installation_dir"
+    echo "Removing $nvm_dir"
+    rm -rf "$nvm_dir"
+    echo "Node.js successfully removed."
 else
     echo "~~~ :node: No Node.js installation to uninstall"
-    echo "The directory is not created by the automattic/nvm-buildkite-plugin: $nvm_installation_dir"
+    echo "Directory does not exist or was not created by this plugin: $nvm_dir"
```

Comment:
Well now that we're using deterministic naming, and that the `if` test back to being only about the dir with deterministic name existing or not…
```suggestion
    echo "Directory not found: $nvm_dir"
```